node.go

```diff
@@ -182,7 +183,7 @@ func NewNode(cli *http.Client, cfg Config) (*Node, error) {
 	mlc.Label = cfg.Label
 
 	if cfg.Log != nil {
-		mlc.LogOutput = log.NewStdlibAdapter(level.Debug(log.With(cfg.Log, "component", "memberlist")))
+		mlc.LogOutput = &memberListOutputLogger{logger: log.With(cfg.Log, "component", "memberlist")}
```

Comment:
As an aside:

This is a mistake I made; I don't think libraries should be determining their own identifying key-value pairs (`component=memberlist`). At some point I'll want to remove the log.With here, but that'll require an Alloy change to move the log.With downstream.

Reply:
Well, in Alloy we pass `service=cluster` and `ckit` logs some of them with just that logger, but some logs that come from `memberlist` get `component=memberlist`. I think it's fine cause `ckit` has two types of logs: its own and ones from its subsystem of memberlist. Maybe the `component=` label is not the best name tho.